<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(136, 128, 159);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>601</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lbl_yisus">
       <property name="styleSheet">
        <string notr="true">image: url(:/equipo/IMG-20240216-WA0026.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_ana">
       <property name="styleSheet">
        <string notr="true">image:url(:/equipo/IMG-20240216-WA0025.jpg)</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_jorge">
       <property name="styleSheet">
        <string notr="true">image: url(:/equipo/IMG-20240216-WA0027.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_axel">
       <property name="styleSheet">
        <string notr="true">image:url(:/equipo/IMG-20240216-WA0028.jpg)</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>601</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="cb_yisus">
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
</string>
       </property>
       <property name="text">
        <string>Yisus</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_Jorge">
       <property name="styleSheet">
        <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Jorge</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_Ana">
       <property name="text">
        <string>Ana Patto</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_axel">
       <property name="text">
        <string>Axel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Equipo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/logos/Unidad2/descarga.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/logos/Unidad2/uat-solo-chico.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>370</y>
      <width>111</width>
      <height>149</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/logos/Unidad2/66d822884e7171e9fb9cfadb8daf80b6.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_equipo">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>330</y>
      <width>104</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../recursos.qrc"/>
 </resources>
 <connections/>
</ui>
